data/src/test/java/org/apache/iceberg/data/DataTest.java

```diff
@@ -31,7 +32,8 @@
 import org.apache.iceberg.types.Types.MapType;
 import org.apache.iceberg.types.Types.StructType;
 import org.junit.Rule;
-import org.junit.Test;
+import org.junit.jupiter.api.Test;
```

Comment:
this class can only be switched to JUnit5 once all of its subclasses are on JUnit5. I see that there are still a few JUnit4 ones in `iceberg-flink`that would need to be converted first